tap_tiktok_ads/streams.py

```diff
@@ -191,6 +191,7 @@ def __init__(self,
         self.state = state
         self.config = config
         self.client = client
+        self.page_size = int(config.get('page_size', 1000))
```

Comment:
Any reason to configure the Page_size?

Reply:
Yes, page size from the Tap is passed as `1000`, but we cannot create more than 999 records for the `campaign` stream. POST call is giving the error like: `"The number of campaigns has reached the maximum limit. Please remove a campaign and try again"`. Also for sandbox, the records for **ad_insights** stream the records are automatically generated. Thus we have added a page size param.